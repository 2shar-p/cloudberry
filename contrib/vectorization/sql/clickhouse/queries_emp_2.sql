-- turn on vectorization

SET vector.enable_vectorization=on;

SET vector.max_batch_size = 128;


-- query 11 -20

SELECT MobilePhoneModel, count(distinct UserID) AS u FROM hits WHERE MobilePhoneModel != '' GROUP BY MobilePhoneModel ORDER BY MobilePhoneModel, u LIMIT 10;

SELECT MobilePhone, MobilePhoneModel, count(distinct UserID) AS u FROM hits WHERE MobilePhoneModel != '' GROUP BY MobilePhone, MobilePhoneModel ORDER BY MobilePhone, MobilePhoneModel, u LIMIT 10;

SELECT SearchPhrase, count() AS c FROM hits WHERE SearchPhrase != '' GROUP BY SearchPhrase ORDER BY SearchPhrase, c LIMIT 10;

SELECT SearchPhrase, count(distinct UserID) AS u FROM hits WHERE SearchPhrase != '' GROUP BY SearchPhrase ORDER BY SearchPhrase, u LIMIT 10;

SELECT SearchEngineID, SearchPhrase, count() AS c FROM hits WHERE SearchPhrase != '' GROUP BY SearchEngineID, SearchPhrase ORDER BY SearchEngineID, SearchPhrase, c LIMIT 10;

SELECT UserID, count() FROM hits GROUP BY UserID ORDER BY userid, count() LIMIT 10;

SELECT UserID, SearchPhrase, count() FROM hits GROUP BY UserID, SearchPhrase ORDER BY UserID, SearchPhrase, count() LIMIT 10;

SELECT UserID, SearchPhrase, count() FROM hits GROUP BY UserID, SearchPhrase LIMIT 10;

SELECT UserID, EventTime AS m, SearchPhrase, count() FROM hits GROUP BY UserID, m, SearchPhrase ORDER BY UserID, m, SearchPhrase, count() LIMIT 10;

SELECT UserID FROM hits WHERE UserID = '1234567890123456789';

