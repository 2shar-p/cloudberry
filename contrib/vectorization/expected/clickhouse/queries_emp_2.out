-- turn on vectorization
SET vector.enable_vectorization=on;
SET vector.max_batch_size = 128;
-- query 11 -20
SELECT MobilePhoneModel, count(distinct UserID) AS u FROM hits WHERE MobilePhoneModel != '' GROUP BY MobilePhoneModel ORDER BY MobilePhoneModel, u LIMIT 10;
 mobilephonemodel | u 
------------------+---
(0 rows)

SELECT MobilePhone, MobilePhoneModel, count(distinct UserID) AS u FROM hits WHERE MobilePhoneModel != '' GROUP BY MobilePhone, MobilePhoneModel ORDER BY MobilePhone, MobilePhoneModel, u LIMIT 10;
 mobilephone | mobilephonemodel | u 
-------------+------------------+---
(0 rows)

SELECT SearchPhrase, count() AS c FROM hits WHERE SearchPhrase != '' GROUP BY SearchPhrase ORDER BY SearchPhrase, c LIMIT 10;
 searchphrase | c 
--------------+---
(0 rows)

SELECT SearchPhrase, count(distinct UserID) AS u FROM hits WHERE SearchPhrase != '' GROUP BY SearchPhrase ORDER BY SearchPhrase, u LIMIT 10;
 searchphrase | u 
--------------+---
(0 rows)

SELECT SearchEngineID, SearchPhrase, count() AS c FROM hits WHERE SearchPhrase != '' GROUP BY SearchEngineID, SearchPhrase ORDER BY SearchEngineID, SearchPhrase, c LIMIT 10;
 searchengineid | searchphrase | c 
----------------+--------------+---
(0 rows)

SELECT UserID, count() FROM hits GROUP BY UserID ORDER BY userid, count() LIMIT 10;
 userid | count 
--------+-------
(0 rows)

SELECT UserID, SearchPhrase, count() FROM hits GROUP BY UserID, SearchPhrase ORDER BY UserID, SearchPhrase, count() LIMIT 10;
 userid | searchphrase | count 
--------+--------------+-------
(0 rows)

SELECT UserID, SearchPhrase, count() FROM hits GROUP BY UserID, SearchPhrase LIMIT 10;
 userid | searchphrase | count 
--------+--------------+-------
(0 rows)

SELECT UserID, EventTime AS m, SearchPhrase, count() FROM hits GROUP BY UserID, m, SearchPhrase ORDER BY UserID, m, SearchPhrase, count() LIMIT 10;
 userid | m | searchphrase | count 
--------+---+--------------+-------
(0 rows)

SELECT UserID FROM hits WHERE UserID = '1234567890123456789';
 userid 
--------
(0 rows)

